lib/Interpreter/CppInterOp.cpp

```diff
@@ -3262,22 +3262,18 @@ namespace Cpp {
     return PI->getNameAsString();
   }
 
-  void GetBinaryOperator(TCppScope_t scope, enum BinaryOperator op,
-                         std::vector<TCppFunction_t>& operators) {
+  void GetOperator(TCppScope_t scope, enum Operator op,
```

Comment:
```suggestion
  void GetOperator(TCppScope_t scope, Operator op,
```

Reply:
And possibly add a `enum OperatorKind { kUnary, kBinary, kBoth}` as a third argument with possibly a `kBoth` as a default value.